test/xunit.analyzers.tests/TheoryMethodMustUseAllParametersTests.cs

```diff
@@ -93,7 +93,7 @@ class TestClass
     [Theory]
     void TestMethod(int foo, int bar, int baz)
     {
-        Console.WriteLine(bar);
+        TestMethod(bar, bar, bar);

```

Comment:
This was necessary since trying to reference `System.Console` created compilation problems. The `Console` type appears to be defined in both `System.Private.CoreLib` and `System.Console`.